js/src/launchAsync.ts

```diff
@@ -116,7 +116,11 @@ export function launchAsync(token: string, subdomain: string, content: Content,
             }
         };
 
-        const parent = options.parent ? options.parent : document.body;
+        options.parent = options.parent === null ? document.body : typeof(options.parent) !== 'undefined' ? options.parent : document.body;
```

Comment:
can't you just check null & undefined in one swoop by checking if truthy?

Reply:
done